### ISSUE TITLE
v1 spec

### ISSUE THREAD

**Issue by stianfro** [stianfro/invoicers#1]
`invoicers --config default.yaml  --file example.yaml` -> generate a markdown with [saphyr](https://github.com/saphyr-rs/saphyr) parsing the yaml-> generate a pdf using pandoc

- default config that contains info that goes on the invoice (yaml?csv?)
  - company name
  - bank info
- invoice file (yaml? csv?)
  - list of products/services with rates
  - on call rate, amount etc
  - customer name & info

Use [pandoc](https://docs.rs/pandoc/latest/pandoc/) to generate pdf?

basically all the options here:
![CleanShot 2024-12-18 at 17 25 43@2x](https://github.com/user-attachments/assets/0125534b-6456-48b4-86a5-5316f15cdc81)

![CleanShot 2024-12-18 at 17 26 17@2x](https://github.com/user-attachments/assets/7ad68840-1699-41fb-a96d-bfa033c4af84)
